cmd/agent/checkers.go

```diff
@@ -68,6 +68,7 @@ func addToMaster(node agent.Agent, config *config, kubeConfig monitoring.KubeCon
 	node.AddChecker(monitoring.DockerHealth(config.dockerAddr))
 	node.AddChecker(etcdChecker)
 	node.AddChecker(monitoring.SystemdHealth())
+	node.AddChecker(monitoring.PingHealth(config.serfRPCAddr,config.role)
```

Comment:
Looks like the closing `)` is missing. Does this compile at all?

Reply:
yeah, fixed... not sure how that got to be missing.